app/src/main/cpp/skyline/common/utils.h

```diff
@@ -20,13 +21,38 @@ namespace skyline::util {
     template<typename T>
     concept TrivialObject = std::is_trivially_copyable_v<T> && !requires(T v) { v.data(); };
 
+    namespace detail {
+        /**
+         * @brief Retrieves the system counter clock frequency
+         * @note Some devices report an incorrect value so they need special handling
+         */
+        inline u64 InitFrequency() {
+            char buffer[PROP_VALUE_MAX];
+            int len{__system_property_get("ro.product.board", buffer)};
+            std::string_view board{buffer, static_cast<size_t>(len)};
+
+            u64 frequency;
+            if (board == "s5e9925")             // Exynos 2200
+                frequency = 25600000;
+            else if (board == "exynos2100")     // Exynos 2100
+                frequency = 26000000;
+            else if (board == "exynos9810")     // Exynos 9810
+                frequency = 26000000;
+            else
+                asm volatile("MRS %0, CNTFRQ_EL0" : "=r"(frequency));
+
+            return frequency;
+        }
+    }
+
+    inline const u64 clockFrequency{detail::InitFrequency()}; //!< The system counter clock frequency in Hz
```

Comment:
Globals should be `CamelCase`, fix all usages of this too.
```suggestion
    inline const u64 ClockFrequency{detail::InitFrequency()}; //!< The system counter clock frequency in Hz
```